packages/framework/src/services/kafka/kafka-broker.client.test.ts

```diff
@@ -0,0 +1,89 @@
+import { Kafka } from 'kafkajs';
+import * as sinon from 'sinon';
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { KafkaBrokerClient, KafkaConsumePayload } from './kafka-broker.client';
+
+const test = suite('Kafka Broker Client');
+
+test.before.each(async (context) => {
+  context.sandbox = sinon.createSandbox();
+});
+
+test.after.each((context) => {
+  context.sandbox.restore();
+});
+
+test('should call connect', async (context) => {
+  const onProducerConnectSpy = context.sandbox.spy(context.sandbox.fake());
+  const onConsumerConnectSpy = context.sandbox.spy(context.sandbox.fake());
+
+  const kafkaProducer = context.sandbox.stub(Kafka.prototype, 'producer').returns({
+    connect: onProducerConnectSpy,
+  });
+  const kafkaConsumer = context.sandbox.stub(Kafka.prototype, 'consumer').returns({
+    connect: onConsumerConnectSpy,
+  });
+
+  const client = new KafkaBrokerClient('localhost:9092');
+  await client.connect();
+
+  assert.ok(kafkaProducer.calledOnce);
+  assert.ok(kafkaConsumer.calledOnce);
+  assert.ok(onProducerConnectSpy.calledOnce);
+  assert.ok(onConsumerConnectSpy.calledOnce);
+});
+
+test('should call subscribe', async (context) => {
+  const payload = {
+    topic: 'test_topic',
+    message: { value: new Buffer('test_value') },
```

Comment:
I'm almost sure `new Buffer()` was deprecated 🤔 we can replace it with:
```suggestion
    message: { value: Buffer.from('test_value', 'utf-8') },
```